Parcare/Services/ParkingLotService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Parking.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Parking.Services
+{
+    public class ParkingLotService
+    {
+        private List<ParkingLotModel> _parkedCars;
+        private int _maxSlots;
+        private int _availableSlots;
```

Comment:
do you need the availableSlots?

Reply:
No, i forgot to decrement it, but i can just modify the method and calculate everytime maxslots - list count